commit 45556c8a5d73beeb2ca79ab995c7a47f1f602bee
Author: Tim Gardner <tim.gardner@canonical.com>
Date:   Thu Sep 17 11:37:43 2015 -0600

    UBUNTU: SAUCE: Add vmlinux.strip to BOOT_TARGETS1
    
    This came in via "UBUNTU: Rebase to v4.2-rc3", but I'm not
    sure why.
    
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/arch/powerpc/Makefile b/arch/powerpc/Makefile
index 1934707..8fe4053 100644
--- a/arch/powerpc/Makefile
+++ b/arch/powerpc/Makefile
@@ -263,7 +263,7 @@ drivers-$(CONFIG_OPROFILE)	+= arch/powerpc/oprofile/
 all: zImage
 
 # With make 3.82 we cannot mix normal and wildcard targets
-BOOT_TARGETS1 := zImage zImage.initrd uImage
+BOOT_TARGETS1 := zImage zImage.initrd uImage vmlinux.strip
 BOOT_TARGETS2 := zImage% dtbImage% treeImage.% cuImage.% simpleImage.% uImage.%
 
 PHONY += $(BOOT_TARGETS1) $(BOOT_TARGETS2)
